src/fsharp/FSharp.Core/FSharp.Core.fsproj

```diff
@@ -4,15 +4,20 @@
 
   <PropertyGroup>
     <OutputType>Library</OutputType>
-    <TargetFrameworks>net45;netstandard2.0</TargetFrameworks>
-    <TargetFrameworks Condition="'$(OS)' == 'Unix'">netstandard2.0</TargetFrameworks>
+    <TargetFrameworks>netstandard2.0</TargetFrameworks>
```

Comment:
Nit: If we're only producing one TFM, use the singular `TargetFramework` property instead of the plural.